app/controllers/nonsipp/memberdetails/SchemeMembersListController.scala

```diff
@@ -180,12 +184,33 @@ class SchemeMembersListController @Inject()(
               }
           }.merge.pure[Future],
           value => {
-            if (lengthOfMembersDetails == maxSchemeMembers && value) {
+            val notChecked = request.userAnswers.get(MembersDetailsChecked(srn)).contains(false)
+
+            if (notChecked) {
```

Comment:
Same to consider here as in above comment. E.g. 
`` if (isPrePopulation && notChecked) ``